scripts/ci/install

```diff
@@ -11,15 +11,17 @@ fi
 wget -qO- "https://raw.githubusercontent.com/travis-ci/artifacts/master/install" |bash
 
 # install Miniconda3
-wget -nv "http://repo.continuum.io/miniconda/${MINICONDA_FILENAME}"
-bash ${MINICONDA_FILENAME} -b -f -p ${HOME}/miniconda
+if  [ "${NO_INSTALL_CONDA}" != "1" ]; then
```

Comment:
wondering if there is a better env var name. `NO_INSTALL_CONDA` says the "what" but not "why" maybe it would be better to have something like `[ "${USER_DOCKER_IMAGE}" != "1" ]`. Not sure

Reply:
I definitely agree "what" is more flexible when many people are using things, possibly for different ends. But in this case the only people interacting with the script are devs and sometimes when the audience and usage is narrow I think the "why" is more important. But I don't have a strong opinion here so happy to leave it as it is.